addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -333,6 +302,74 @@
                 </div>
             </div>
         </t>
+        <t t-elif="channel.enroll in ['public', 'invite']" name="o_wslides_prerequisite_test">
+            <div t-if="channel.prerequisite_channel_ids and not channel.has_completed_prerequisites"
+                 class="text-center">
+                <div class="alert my-0 bg-100">
+                    <h6>
+                        <i class="fa fa-lock"/>
+                        <span>Course Locked</span>
+                    </h6>
+                    <div>
+                        <small>
+                            Finish
+                            <a t-if="len(channel.prerequisite_channel_ids) == 1"
+                               t-attf-href="/slides/{{slug(channel.prerequisite_channel_ids)}}">
+                                <span t-out="channel.prerequisite_channel_ids.name"/>
```

Comment:
Even if 100% the same I would prefer "channel.prerequisite_channel_ids[0].name".

Also I don't think ".name" is necessary, as t-outing a recordset should show its display name?

(Also maybe the t-out can be inside the a? Trying to gain some indentation levels ^^)